wrangler-core/src/main/java/io/cdap/directives/row/Fail.java

```diff
@@ -106,4 +109,13 @@ public Mutation lineage() {
     el.variables().forEach(col -> builder.relation(col, col));
     return builder.build();
   }
+
+  @Override
+  public List<WranglerMetric> getMetrics() {
+    WranglerMetric jexlCategoryMetric = getJEXLCategoryMetric(el.script.getParsedText());
+    if (jexlCategoryMetric != null) {
+      return ImmutableList.of(jexlCategoryMetric);
```

Comment:
nit:
` if (jexlCategoryMetric != null) {
      return ImmutableList.of(jexlCategoryMetric);
    }
    return ImmutableList.of();` 
    replace with single statement:
`return (jexlCategoryMetric == null) ? ImmutableList.of() : ImmutableList.of(jexlCategoryMetric);`

Reply:
Done